src/payment/models/crypto-input/btc-input.service.ts

```diff
@@ -177,6 +179,7 @@ export class BtcInputService extends CryptoInputService {
           const { confirmations } = await this.btcClient.waitForTx(input.outTxId);
           if (confirmations > 60) {
             await this.cryptoInputRepo.update(input.id, { isConfirmed: true });
+            await this.buyCryptoService.createFromCrypto(input.id, input.route.id);
```

Comment:
Talked to Robin, he wants the buyCrypto to be created directly after registration of the input (after line 63).